<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMS</class>
 <widget class="QMainWindow" name="SMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Sort</string>
     </property>
     <addaction name="actionSortID"/>
     <addaction name="actionSortName"/>
    </widget>
    <addaction name="actionAdd"/>
    <addaction name="actionAlter"/>
    <addaction name="actionDelete"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionOriginQss"/>
     <addaction name="actionQssBlue"/>
    </widget>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatus"/>
    <addaction name="menu_7"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Window(&amp;W)</string>
    </property>
    <addaction name="actionTabBar"/>
    <addaction name="actionWindow"/>
    <addaction name="actionSascadeSubWindows"/>
    <addaction name="actionTileSubWindows"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrint"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionAlter"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSort"/>
   <addaction name="actionFilter"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/new.png</normaloff>:/icon/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/open.png</normaloff>:/icon/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/close.png</normaloff>:/icon/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>Close(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>增加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/about.png</normaloff>:/icon/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/save.png</normaloff>:/icon/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAlter">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/alert.png</normaloff>:/icon/icon/alert.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/delete.png</normaloff>:/icon/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSortID">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/sort.png</normaloff>:/icon/icon/sort.png</iconset>
   </property>
   <property name="text">
    <string>By ID</string>
   </property>
   <property name="toolTip">
    <string>按学号排序</string>
   </property>
  </action>
  <action name="actionSortName">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/sort.png</normaloff>:/icon/icon/sort.png</iconset>
   </property>
   <property name="text">
    <string>By Name</string>
   </property>
   <property name="toolTip">
    <string>按姓名排序</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/print.png</normaloff>:/icon/icon/print.png</iconset>
   </property>
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/saveAs.png</normaloff>:/icon/icon/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save As...(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionTabBar">
   <property name="text">
    <string>Label Mode</string>
   </property>
   <property name="toolTip">
    <string>视窗标签模式</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>Window Mode</string>
   </property>
   <property name="toolTip">
    <string>视窗窗口模式</string>
   </property>
  </action>
  <action name="actionSascadeSubWindows">
   <property name="text">
    <string>Overlapping</string>
   </property>
   <property name="toolTip">
    <string>重叠排列</string>
   </property>
  </action>
  <action name="actionTileSubWindows">
   <property name="text">
    <string>Neatly</string>
   </property>
   <property name="toolTip">
    <string>整齐排列</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/sort.png</normaloff>:/icon/icon/sort.png</iconset>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
   <property name="toolTip">
    <string>排序</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/search .png</normaloff>:/icon/icon/search .png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
   <property name="toolTip">
    <string>显示状态栏</string>
   </property>
  </action>
  <action name="actionOriginQss">
   <property name="text">
    <string>Original</string>
   </property>
   <property name="toolTip">
    <string>原生皮肤</string>
   </property>
  </action>
  <action name="actionQssBlue">
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="toolTip">
    <string>蓝色主题</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
